SpringProject/robot_solver_split_controller_V2.m

```diff
@@ -97,6 +97,7 @@
 Kp_floor = -5000; % floor spring constant
 Kd_floor = -800; % floor damping coefficient
 b_fric_floor = -2000; % floor coefficient of lateral, viscous friction
+state = zeros(1, 36);
```

Comment:
bazinga

Reply:
fixed in 5a5ef73d32e69b13da3e458b634e3e16500b227e